src/CopyrighFixer/CopyrighFixer/bar.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BAR_H
+#define BAR_H
+
+#include <QString>
+#include <iostream>
+
+class Bar
+{
+public:
+    Bar();
+    QString bar = "Hello WORLD!";
+
+QString foo(){
```

Comment:
You need move implementation of tour all methods into cpp file

Reply:
Done